bank/src/main/java/org/example/Bank/CentralBank.java

```diff
@@ -0,0 +1,36 @@
+package org.example.Bank;
+
+
+import org.example.Account.Account;
+
+import java.util.ArrayList;
+
+public class CentralBank {
+
+
+    //은행 계좌들을 관리하는 중앙은행 클래스
+    private static CentralBank instance = new CentralBank();
```

Comment:
```suggestion
    private static final CentralBank instance = new CentralBank();
```
final 키워드를 사용하여 변경되지 않을 인스턴스라는 것을 명시해야합니다